Smartling.API/Job/JobApiClient.cs

```diff
@@ -78,35 +78,42 @@ public virtual AddFileToJobResponse AddLocale(string localeId, string jobId, boo
       return JsonConvert.DeserializeObject<AddFileToJobResponse>(response["response"]["data"].ToString());
     }
 
-    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "")
+    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "", string statusFilter = "")
     {
-      var page = GetPage(jobName, PageSize, 0);
+      var page = GetPage(jobName, PageSize, 0, statusFilter);
       var results = new List<Smartling.Api.Model.Job>();
       var pageNumber = 0;
       results.AddRange(page.items);
 
       while (page.totalCount > results.Count && page.items.Count > 0)
       {
         pageNumber++;
-        page = GetPage(jobName, PageSize, PageSize * pageNumber);
+        page = GetPage(jobName, PageSize, PageSize * pageNumber, statusFilter);
         results.AddRange(page.items);
       }
 
       return results;
     }
 
-    public virtual JobList GetPage(string jobName, int limit, int offset)
+    public virtual List<Smartling.Api.Model.Job> GetActive()
```

Comment:
Well, maybe it's useful to have a predefined filter in SDK but I believe every developer has their own opinion on what is "active" job. The second reason why I dislike this change this is how `ActiveJobsFilter` const looks. It takes time to figure out how it works.
Let's leave the new `Get(string jobName = "", string statusFilter = "")` and move filter string and decision to sitecore connector.

Reply:
Ok, done.